src/config/configJsonSchema.js

```diff
@@ -36,13 +39,47 @@ module.exports = {
           type: 'object',
           properties: {
             port: {
-              type: 'integer',
-              minimum: 0,
+              $ref: '#/definitions/port',
+            },
+            seeds: {
+              type: 'array',
+              items: {
+                type: 'object',
+                properties: {
+                  host: {
+                    type: 'string',
+                  },
+                  port: {
+                    $ref: '#/definitions/port',
+                  },
+                },
+                required: ['host', 'port'],
+                additionalProperties: false,
+              },
             },
           },
-          required: ['port'],
+          required: ['port', 'seeds'],
           additionalProperties: false,
         },
+        rpc: {
+          type: 'object',
+          properties: {
+            port: {
+              $ref: '#/definitions/port',
+            },
+            user: {
+              type: 'string',
```

Comment:
maxLength: 1

Reply:
Why? The RPC username is usually more than 1 character.

Reply:
I meant minLength, sorry.